src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -4168,6 +4168,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncIteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Diagnostics_CodeAnalysis_UnscopedRefAttribute__ctor:
```

Comment:
>WellKnownMember.System_Diagnostics_CodeAnalysis_UnscopedRefAttribute__ctor [](http://example.com/codeflow?start=21&length=74)

I am curious what scenarios does this enable. Is presence of the attribute not important for consumers?

Reply:
It looks like we are not testing effect of this change. Also, I think that, unless we have a compelling scenario that we would like to enable, it might be better to detect the lack of the API and report an error instead of silently dropping the attribute. We already do this for default value and param array synthesized attributes.